lesson_03/homework_03.py

```diff
@@ -0,0 +1,173 @@
+# task 01 == Розділіть змінну alice_in_wonderland так, щоб вона займала декілька фізичних лінії
+# task 02 == Знайдіть та відобразіть всі символи одинарної лапки (') у тексті
+alice_in_wonderland = '"Would you tell me, please, which way I ought to go from here?"\n'\
```

Comment:
You have tried using \n symbol find all ' as I understand. But for task 02 can only do like ->
`for symbol in alice_in_wonderland:
      if symbol == "'": 
          print(symbol)`
Or sth like that but in this way you are able to find and show all ' symbols.

Reply:
Updated task_02 in the commit cf9e13f84c6c588b5f0cd2707e4f3e51c347085c